src/Products/PlonePAS/sheet.py

```diff
@@ -8,6 +8,7 @@
 from Products.CMFCore.interfaces import ISiteRoot
 from Products.PlonePAS.interfaces.propertysheets import IMutablePropertySheet
 from Products.PluggableAuthService.UserPropertySheet import UserPropertySheet
+from past.builtins import long
```

Comment:
We are Python 3.6+ so I doubt it is needed.

Reply:
Ditto.